modules/webgl/src/adapter/converters/device-parameters.ts

```diff
@@ -169,9 +181,9 @@ export function setDeviceParameters(device: Device, parameters: Parameters) {
     const ext = extensions.WEBGL_polygon_mode;
 
     if (parameters.polygonMode) {
-      const mode = map('polygonMode', parameters.provokingVertex, {
+      const mode = map<PolygonMode, GLPolygonMode>('polygonMode', parameters.polygonMode, {
```

Comment:
I guess we could alternatively just declare the result variable type 'constant mode: polygonmode = map...' to to force a check

Reply:
Yes – I think that approach would validate `V` for our `Record<K, V>` mapping. But alone, it wouldn't tell us whether we've covered every possible `K`.